Tests/ProGetAutomationTest/ProGetAutomationTest.psm1

```diff
@@ -189,7 +189,7 @@ try
     $activateUri = New-Object 'Uri' $uri,'/administration/licensing/activate'
     Invoke-WebRequest -Uri $activateUri -WebSession $activationWebSession
 
-    $tasksUri = [Uri]::New($uri, '/administration/security/tasks')
+    $tasksUri = [Uri]::New($uri, '/administration/security/tasks61')

```

Comment:
Is this really the path to this page?

Reply:
At least in 6.0.18 (maybe down to .13) as well. 

![image](https://user-images.githubusercontent.com/1430176/182240021-fee2fffc-c534-4048-8658-64c18ce178e2.png)

Reply:
Appear to be a 6.1 Preview feature. "revert to single-directory mode" adds back the "/task" endpoint. Chance since at least 6.0.13 with default install.